src/driver/verify.cpp

```diff
@@ -74,6 +75,7 @@ verify::tolerance get_tolerances(const program& p,
 
 std::vector<argument> run_ref(program p, const parameter_map& inputs)
 {
+    fp_to_double(p);
```

Comment:
Do we want to do this unconditionally?

Reply:
We could add an option to the driver, but then need another flag to be passed.